--- torchwood.debianize/debian/changelog	2026-06-10 14:51:19.686170882 +0530
+++ torchwood/debian/changelog	2026-06-10 14:51:28.460940718 +0530
@@ -1,5 +1,33 @@
-golang-github-FiloSottile-torchwood (0.9.0-1) UNRELEASED; urgency=low
+torchwood (0.9.0-5) unstable; urgency=medium
 
-  * Initial release.
+  * Add Breaks/Replaces/Conflicts on litetlog (Closes: #1134632)
+  * Rename /usr/bin/spicy to spicy-torchwood (Closes: #1134629)
 
- -- Anu-Ra-g <nayakanurag078@gmail.com>  Wed, 10 Jun 2026 14:51:19 +0530
+ -- Simon Josefsson <simon@josefsson.org>  Mon, 11 May 2026 10:56:12 +0200
+
+torchwood (0.9.0-4) unstable; urgency=medium
+
+  * Extend timeout 25m->50m for riscv64
+
+ -- Simon Josefsson <simon@josefsson.org>  Sun, 19 Apr 2026 09:10:09 +0200
+
+torchwood (0.9.0-3) unstable; urgency=medium
+
+  * Standards-Version: 4.7.4
+  * Use go.mod vrf.go vendor commit hash
+  * Timeout 10m->25m for riscv64
+  * Silence lintian privacy-breach-generic
+
+ -- Simon Josefsson <simon@josefsson.org>  Thu, 16 Apr 2026 18:03:39 +0200
+
+torchwood (0.9.0-2) unstable; urgency=medium
+
+  * Silence Salsa lintian
+
+ -- Simon Josefsson <simon@josefsson.org>  Mon, 16 Mar 2026 08:18:03 +0100
+
+torchwood (0.9.0-1) unstable; urgency=medium
+
+  * Initial release (Closes: #1111262)
+
+ -- Simon Josefsson <simon@josefsson.org>  Sun, 15 Mar 2026 20:18:45 +0100
