--- prometheus-redis-exporter.debianize/debian/control	2026-06-10 14:53:12.252744732 +0530
+++ prometheus-redis-exporter/debian/control	2026-06-10 14:53:18.088489513 +0530
@@ -1,15 +1,33 @@
-Source: golang-github-oliver006-redis-exporter
-Section: devel
-Maintainer: Anu-Ra-g <nayakanurag078@gmail.com>
-Build-Depends: debhelper-compat (= 13), dh-sequence-golang, golang-github-beorn7-perks-dev, golang-github-gomodule-redigo-dev, golang-github-klauspost-compress-dev, golang-github-kr-text-dev, golang-github-mna-redisc-dev, golang-github-munnerz-goautoneg-dev, golang-github-prometheus-client-golang-dev, golang-github-prometheus-client-model-dev, golang-github-prometheus-common-dev, golang-github-prometheus-procfs-dev, golang-github-sirupsen-logrus-dev, golang-go (>= 1.25.0), golang-golang-x-crypto-dev, golang-golang-x-sys-dev, golang-google-protobuf-dev, golang-gopkg-yaml.v2-dev
-Standards-Version: 4.7.4.1
-Vcs-Browser: https://github.com/oliver006/redis_exporter,branch=debian%2Fmain
-Vcs-Git: https://github.com/oliver006/redis_exporter,branch=debian%2Fmain.git
-Rules-Requires-Root: no
+Source: prometheus-redis-exporter
+Section: net
+Maintainer: Debian Go Packaging Team <team+pkg-go@tracker.debian.org>
+Uploaders:
+ Guillem Jover <gjover@sipwise.com>,
+Build-Depends:
+ debhelper-compat (= 13),
+ dh-sequence-golang,
+ golang-any (>= 2:1.20~),
+ golang-github-gomodule-redigo-dev (>= 1:1.8.3-1~),
+ golang-github-mna-redisc-dev,
+ golang-github-prometheus-client-golang-dev,
+ golang-github-prometheus-client-model-dev,
+ golang-github-sirupsen-logrus-dev,
 Testsuite: autopkgtest-pkg-go
+Standards-Version: 4.7.4
+Vcs-Browser: https://salsa.debian.org/go-team/packages/prometheus-redis-exporter
+Vcs-Git: https://salsa.debian.org/go-team/packages/prometheus-redis-exporter.git
+Homepage: https://github.com/oliver006/redis_exporter
 XS-Go-Import-Path: github.com/oliver006/redis_exporter
 
-Package: golang-github-oliver006-redis-exporter-dev
-Architecture: all
-Multi-Arch: foreign
-Depends: ${misc:Depends}
+Package: prometheus-redis-exporter
+Architecture: any
+Pre-Depends:
+ ${misc:Pre-Depends},
+Depends:
+ adduser,
+ ${misc:Depends},
+ ${shlibs:Depends},
+Static-Built-Using:
+ ${misc:Static-Built-Using},
+Description: Prometheus exporter for Valkey metrics (Redis-compatible)
+ Supports Valkey and Redis 2.x, 3.x, 4.x, 5.x, 6.x, and 7.x.
